src/js/components/TaskFileListComponent.jsx

```diff
@@ -0,0 +1,179 @@
+var React = require("react/addons");
+var classNames = require("classnames");
+var lazy = require("lazy.js");
+
+var Util = require("../helpers/Util");
+var MesosActions = require("../actions/MesosActions");
+var MesosEvents = require("../events/MesosEvents");
+var MesosStore = require("../stores/MesosStore");
+
+var TaskFileListComponent = React.createClass({
+  displayName: "TaskFileListComponent",
+  propTypes: {
+    task: React.PropTypes.object.isRequired
+  },
+
+  getInitialState: function () {
+    var task = this.props.task;
+    var taskId = task.id || task.taskId;
+    return {
+      files: MesosStore.getTaskFiles(taskId)
+    };
+  },
+
+  componentWillMount: function () {
```

Comment:
[super nitpick mode on]
`componentDidMount` should come first
[super nitpick mode off]